### DISCUSSION TITLE
WPF newest schema

### DISCUSSION THREAD

**Discussion by Duf1220** [dotnet/wpf#7889] [General]
Hi. What is the newest schema used for wpf? Is it "http://schemas.microsoft.com/winfx/2009/xaml"?

**Comment by miloush** [comment]
That is a slightly confusing question, you are asking about schema but listed a namespace. I believe the namespace you suggested is not used by WPF, rather it is used by Xamarin/MAUI: https://learn.microsoft.com/en-us/dotnet/maui/xaml/namespaces/. WPF uses http://schemas.microsoft.com/winfx/2006/xaml, see https://learn.microsoft.com/en-us/dotnet/desktop/wpf/advanced/xaml-namespaces-and-namespace-mapping-for-wpf-xaml.
